lib/SPIRV/SPIRVUtil.cpp

```diff
@@ -484,6 +491,21 @@ bool oclIsBuiltin(StringRef Name, StringRef &DemangledName, bool IsCpp) {
   return false;
 }
 
+// Demangled name is a substring of the name. The DemangledName is updated only
+// if true is returned
```

Comment:
I think it should be moved to the description, where the function is declared.
```suggestion
// DemangledName is a substring of the Name. The DemangledName is updated only
// if true is returned
```